### ISSUE TITLE
Updating documentation for OpenType related informations

### ISSUE THREAD

**Issue by KrasnayaPloshchad** [typiconman/fonts-cu#27]
LibreOffice 5.3 released in Feb 2017, which give users an ability to access OpenType features, and making Graphite layout works on macOS, so the documention should be updated to reflect such changes.

**Comment by typiconman** [comment]
I updated the doc. @KrasnayaPloshchad, please comment.
For convenience, the latest version is attached.
[fonts-churchslavonic.pdf](https://github.com/typiconman/fonts-cu/files/924445/fonts-churchslavonic.pdf)

**Comment by KrasnayaPloshchad** [comment]
Yes, I reviewed them, and I feel satisfied. Additionaly, I also found LibreOffice have a environment variable can be used to disable Graphite layout: `SAL_DISABLE_GRAPHITE` (see [TDF Wiki](https://wiki.documentfoundation.org/Development/Environment_variables) and [SIL website](http://scripts.sil.org/cms/scripts/page.php?site_id=projects&item_id=graphite_apps)), can you add the instruction into section 9?

**Comment by typiconman** [comment]
I tried setting `SAL_DISABLE_GRAPHITE` on Linux Mint, but it does not seem to do anything with LibreOffice 5.3. Is it only supported in Windows?

**Comment by KrasnayaPloshchad** [comment]
Sorry, I don’t know how to set that.

**Comment by KrasnayaPloshchad** [comment]
Sorry, I have no idea how to use it on Windows.

**Comment by KrasnayaPloshchad** [comment]
I have found MS Office allowing stylisitc sets, see [this attachment](https://bugs.documentfoundation.org/attachment.cgi?id=73540&action=edit) at TDF Bugzilla.

**Comment by typiconman** [comment]
Unfortunately, I have no way to test MS Office.

Regarding the `SAL_DISABLE_GRAPHITE` environmental variable, since I cannot get it to do anything, I don't see a need to mention it in the doc. Perhaps since LO has transitioned to using HarfBuzz, this is not supported anymore?

**Comment by typiconman** [comment]
I added a mention of MS Word in commit df5dcdcef7a1dc37410ff1960276a02f07796b2d. I hope it is correct, since I have no way to test.

**Comment by KrasnayaPloshchad** [comment]
That’s right. I have used MS Office 2013 several years ago, and I have used Font Feature options, they works well for me after I added English languge pack. But I have no longer used it and later I uninstalled it over 1 years ago since it become extremely huge for me (it takes nearly half of my C: drive after several updates from Windows Update, even if I used CCleaner), so I can only help you here, sorry.
